store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9623,4 +9623,14 @@ TODO: delete them permanently from here
     for all unconfigured host names. See also related attributes 'zimbraVirtualHostname' and 'zimbraVirtualIPAddress'.
   </desc>
 </attr>
+
+<attr id="5002" name="zimbraSmtpStartTlsMode" type="enum" value="on,off,only" cardinality="single" optionalIn="globalConfig,server,domain" flags="serverInherited" since="8.8.6">
+  <globalConfigValue>only</globalConfigValue>
+  <globalConfigValueUpgrade>off</globalConfigValueUpgrade>
+  <desc>Configures the starttls mode on sending messages from mailboxd to MTA.
+        ( on   - uses starttls if the peer MTA supports.
```

Comment:
nit: parenthesis is not used in desc of other attrs like zimbraAccountStatus and zimbraReverseProxyImapStartTlsMode.

Reply:
nit: although we're inconsistent with this, I'd also suggest removing periods at end of statements following the hyphen (-)